examples/blogs__use-react-devtools/4.2.1_0/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+   "background": {
+      "persistent": false,
+      "scripts": [ "build/background.js" ]
+   },
+   "browser_action": {
+      "default_icon": {
+         "128": "icons/128-disabled.png",
+         "16": "icons/16-disabled.png",
+         "32": "icons/32-disabled.png",
+         "48": "icons/48-disabled.png"
+      },
+      "default_popup": "popups/disabled.html"
+   },
+   "content_scripts": [ {
+      "js": [ "build/injectGlobalHook.js" ],
+      "matches": [ "\u003Call_urls>" ],
+      "run_at": "document_start"
+   } ],
+   "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'",
+   "description": "Adds React debugging tools to the Chrome Developer Tools.\n\nCreated from revision 3816ae7c3 on 11/27/2019.",
+   "devtools_page": "main.html",
+   "icons": {
+      "128": "icons/128-production.png",
+      "16": "icons/16-production.png",
+      "32": "icons/32-production.png",
+      "48": "icons/48-production.png"
+   },
+   "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA39Lr82J1eer+RvXeQG6HYHb2A6xU/83bcIyOcu+Wu2vYO/jbBvlD3m/xqrU4xSXJ/uAC1XY/YtlRs2X2RE1XiYMMlnqw27QdI5tVcSnKedmTFNAsnSaQ/OZT5tzH3HntLDfrFof5BC5RmFuie+kQKGTh673JW0ZdKTUsW406Fckagl9WpPL+qp3Gah3Jw02zxfQX9yUd5gJHYvv8T31DO9+HsI8lPeEjanURY59rRUfg3JKy1aPfts6eISy9kpHC+LrRSHU2Zp5AuUkOh8qEg6DoHFwBNYX20bdd6ekj0OZiIz/9xj9ceNhAr85QYB0641Tzhs3Tg96k56VNwjdN2QIDAQAB",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded AWS Secret Access Key ID. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources that use this access key and using IAM roles instead of hardcoded access keys for IAM users. If you must use IAM users, secure their secrets outside your code.

[Learn more about the use of hardcoded credentials](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html)